test/clj_foundation/config_test.clj

```diff
@@ -2,9 +2,10 @@
   (:require [clojure.test :refer :all]
             [clojure.java.io :as io]
             [schema.core :as s :refer [=> =>*]]
+            [clj-foundation.unit-test-common :as common]
             [clj-foundation.config :refer :all]))
 
-(s/set-compile-fn-validation! true)
+(common/register-fixtures)
 
```

Comment:
Migrated all test namespaces to (common/register-fixtures) rather than ad hoc initialization as before.